azure-pipelines.yaml

```
@@ -35,11 +35,11 @@ pool:
  
 variables:
   - name: sonarqubeProject
-    value: "" #update the sonarqube project for the respective repos
-  - group: "cjs-common-platform" #this is common for all the repos
- 
+    value: "uk.gov.justice:cp-maven-super-pom"
+  - group: "cjs-common-platform"
+
 stages:
-  - template: pipelines/cjscp-framework-build.yaml@cppAzureDevOpsTemplates
+  - template: pipelines/cp-framework-build.yaml@cppAzureDevOpsTemplates
```

```suggestion
  - template: pipelines/cjscp-framework-build.yaml@cppAzureDevOpsTemplates
```

Leave this for now as the ADO templates haven't been updated yet, I'll do this tomorrow